### DISCUSSION TITLE
Inlet service_use-src-addr-for-exporter-addr

### DISCUSSION THREAD

**Discussion by linhnt-hub** [akvorado/akvorado#1568] [Q&A]
Hi guy, 
I'm trying undestand about option use-src-addr-for-exporter-addr on inlet service config:
As doc, With use-src-addr-for-exporter-addr set to true, the source ip of the received flow packet is used as exporter address.
But when I change it to use-src-addr-for-exporter-addr = false, It still same ip address of exporter. 
Do you recommend for me about this option? and which default value of it?
Thanks

flow:
  inputs:
    - type: udp
      decoder: netflow
      listen: :2055
      workers: 3
      use-src-addr-for-exporter-addr: false
    - type: udp
      decoder: sflow
      listen: :6343
      workers: 3
  workers: 2

**Comment by vincentbernat** [comment]
The alternative is to use the address embedded within the flow. Maybe it's the same in your case?

**Reply by linhnt-hub** [reply]
Yes, I saw it's the same in my case.
That mean when I change to "false". Inlet will extract info of exporter from netflow header, right?

**Reply by vincentbernat** [reply]
Yes

**Reply by linhnt-hub** [reply]
Thanks Vincentbernat
But I have one more concern about info of export address in header? Could you told me which field in netflow header akvorado get for exporter ip address (with case use-src-addr-for-exporter-addr= false)?

**Reply by vincentbernat** [reply]
Sorry, for NetFlow, this the same. For sFlow, the agent IP is encoded inside the sflow packet.
